shared/src/persistence/dynamo/notifications/deleteUserConnection.js

```diff
@@ -17,6 +17,13 @@ exports.deleteUserConnection = async ({ applicationContext, connectionId }) => {
     applicationContext,
   });
 
+  if (!connection) {
+    // it must have expired
```

Comment:
Nit: Could this comment be added to that first test name instead?

Reply:
done!